docs/configuration.md

```diff
@@ -56,40 +56,6 @@ spec:
     storage:
       create_default_storage_class: false
       type: external_storage
-  images:
```

Comment:
Are there any other places in the docs that list all the images that k0s will use? I wonder if it would make sense to have _some_ place in the docs where they're listed.